### PR TITLE
Update administration-guide.html.md

### PR DESCRIPTION
**Pull request by aaviram**
"oVirt Node" can be confusing and imply that we talk about an oVirt's host. This can make users and developers think that local storage can be set only on "/data/images" which is wrong.

### CODE REVIEW

File: source/documentation/admin-guide/administration-guide.html.md [hunk 1]
```diff
@@ -2890,7 +2890,7 @@ The new NFS data domain is displayed on the **Storage** tab with a status of `Lo
 A local storage domain can be set up on a host. When you set up host to use local storage, the host automatically gets added to a new data center and cluster that no other hosts can be added to. Multiple host clusters require that all hosts have access to all storage domains, which is not possible with local storage. Virtual machines created in a single host cluster cannot be migrated, fenced or scheduled.
 
 <div class="alert alert-info">
-**Important:** On oVirt Nodes the only path permitted for use as local storage is `/data/images`. This directory already exists with the correct permissions on Hypervisor installations. The steps in this procedure are only required when preparing local storage on Red Hat Enterprise Linux virtualization hosts.
+**Important:** On the "oVirt Node" system the only path permitted for use as local storage is `/data/images`. This directory already exists with the correct permissions on Hypervisor installations. The steps in this procedure are only required when preparing local storage on Red Hat Enterprise Linux virtualization hosts.
```
Comment (mureinik) [inline source/documentation/admin-guide/administration-guide.html.md]: The last sentence is obviously copied from the RHEV documentation, and needs to be amended for use in oVirt - how about "The steps in this procedure are only required when preparing local storage on an oVirt hypervisor installed on a stock OS, such as Fedora or Centos".

Comment (thatdocslady) [inline source/documentation/admin-guide/administration-guide.html.md]: +1 @mureinik @aaviram

**Comment by aaviram** [comment]
The fix is related to the phrasing of the first sentence, I think changing the last sentence should be addressed in another patch. @mureinik

**Comment by mureinik** [comment]
wrt: https://github.com/oVirt/ovirt-site/pull/66#issuecomment-185677602 I'm fine with having the fix I suggested in another patch, just make sure you get them both :-)

**Comment by aaviram** [comment]
@mureinik next commit in the chain adds the text you are suggesting

**Comment by mureinik** [comment]
+1

**Pull request merged**